e2e/fuzz/runner.go

```diff
@@ -12,6 +12,7 @@ import (
 
 var (
 	revertProbabilityThreshold = 20
+	waitForHeightTimeInterval  = 3 * time.Minute
```

Comment:
Camel case notation (note that I haven't been able to change `revertProbabilityThreshold` reference, because it was not added through this PR).
```suggestion
	RevertProbabilityThreshold = 20
	WaitForHeightTimeInterval  = 3 * time.Minute
```